README.md

```diff
@@ -142,8 +142,9 @@ Your output should be similar to the one below:
 Run the following command in terminal:
 ```shell
 terrahub configure -i google_storage -c component.template.terraform.backend.local.path='/tmp/.terrahub/local_backend/google_storage/terraform.tfstate'
-terrahub configure -i google_storage -c component.template.resource.google_storage_bucket.google_storage.name='demo_terraform_automation_${local.project["code"]}'
+terrahub configure -i google_storage -c component.template.resource.google_storage_bucket.google_storage.name='google_storage_99999999'
```

Comment:
let's use `$GOOGLE_STORAGE_BUCKET` as variable instead of hardcoding the value `google_storage_9999999`